README.md

```diff
@@ -12,46 +12,58 @@ oc process -f https://raw.githubusercontent.com/jaegertracing/jaeger-openshift/m
 Once everything is ready, `oc status` tells you where to find Jaeger URL.
 
 ## Production setup
-This template deploys all Jaeger components as a separate services: StatefulSet Cassandra storage, agent as DaemonSet,
-collector and query service with UI. Each one of those can be managed and scaled individually. Because this template
-deploys the agent as a DaemonSet it requires system:admin permissions, therefore cannot be used in OpenShift online.
+This template deploys all Jaeger components as a separate services: StatefulSet Cassandra storage and a service for the Collector
```

Comment:
Minor changes: "This template deploys the Collector, Query Service (with UI) and Cassandra storage (StatefulSet) as separate individually scalable services."